src/openforms/config/models/config.py

```diff
@@ -321,15 +321,18 @@ class GlobalConfiguration(SingletonModel):
     )
 
     # global payment settings
-    payment_order_id_prefix = models.CharField(
-        _("Payment Order ID prefix"),
-        max_length=16,
-        default="{year}",
-        blank=True,
-        help_text=_(
-            "Prefix to apply to generated numerical order IDs. Alpha-numerical only, supports placeholder {year}."
-        ),
-        validators=[validate_payment_order_id_prefix],
+    payment_order_id_template = models.CharField(
+        _("Payment Order ID template"),
+        max_length=48,
```

Comment:
what's the reason for this magic number?

Reply:
It's a reasonable to default, not too long to avoid having runtime errors with ogone

Reply:
I'm trying to understand it, because the placeholder tokens will shrink/expand depending on the values

e.g. `{year}` will always be 4 chars, but `{public_reference}` can be a long `ZAAK-123456789` or short `OF-123456` number, while `uid` will realistically take 1-5 chars. But the tokens/placeholders don't cleanly map to lengths of their values, so I think we can't really make any assumptions/guarantees about avoiding runtime Ogone errors.

Reply:
Yeah I was thinking the same it's a bit hard to manage. Should we go with 255?

Reply:
no need, we can increase it if it's too small - just add a comment that there's no particular reason for 48 and that the template substitutions make this tricky anyway.